src/pages/study-room/util/studyStep.ts

```diff
@@ -0,0 +1,16 @@
+export const studyStep = (type: string) => {
```

Comment:
step으로 표현되는게 조금 더 이해하기 편할 것 같네요~
```suggestion
export const studyStep = (step: string) => {
```

Reply:
함수의 동작을 보면 대략 이런식으로 표현되어야 하지 않을까요?
```suggestion
export const getNextStudyStep = (type: string) => {
```

Reply:
해당 함수는 스터디룸의 스텝을 구하는 함수인데 유틸로 보기엔 어려울 것 같아요.
도메인 로직이 아닐까요?